config/7.1.0/securebanking/ig/scripts/groovy/RepoConsent.groovy

```diff
@@ -130,11 +116,11 @@ def convertIDMResponse(intentResponseObject, intentType) {
         case IntentType.PAYMENT_DOMESTIC_CONSENT:
             responseObj = [
                     "id"                   : intentResponseObject._id,
-                    "data"                 : intentResponseObject.Data,
+                    "data"                 : intentResponseObject.OBIntentObject.Data,
                     // RS expect 'Data' instead 'data' to deserialize the json to OB object
```

Comment:
I suspect that keep the duplication of `d/Data` is to keep the compatibility until finished the epic task, right?

Reply:
Yes, I will review the RCS code and clean this up afterwards. If we can get this set of changes merged then the next payment APIs can build on top of it, this cleanup can then be done without impacting anyone I think